schema.sql

```diff
@@ -12,5 +12,23 @@ CREATE TABLE animals (
 ALTER TABLE animals
   ADD species text;
 
+CREATE TABLE owners (id int GENERATED ALWAYS AS IDENTITY, full_name text, age int, PRIMARY KEY(id));
 
+CREATE TABLE species (id int GENERATED ALWAYS AS IDENTITY, name text, PRIMARY KEY(id));
 
+ALTER TABLE animals ADD COLUMN id SERIAL PRIMARY KEY;
+
+ALTER TABLE animals 
+DROP COLUMN species;
+
+ALTER TABLE animals
+  ADD species_id int;
+
+ALTER TABLE animals
+ADD FOREIGN KEY (species_id) REFERENCES species (id);
+
+ALTER TABLE animals
+  ADD owner_id int;
+
+ALTER TABLE animals
+ADD FOREIGN KEY (owner_id) REFERENCES owners (id);
```

Comment:
✅ Remove column `species` 👍🏻 
✅ Add column `species_id` which is a foreign key referencing `species` table 👍🏻 
✅ Add column `owner_id` which is a foreign key referencing the `owners` table 👍🏻